lib/ask/runtime/flow.ex

```diff
@@ -40,10 +40,20 @@ defmodule Ask.Runtime.Flow do
 
   defp next_step_by_skip_logic(flow, step, reply_value) do
     skip_logic =
-      step
-      |> Map.get("choices")
-      |> Enum.find(fn choice -> choice["value"] == reply_value end)
-      |> Map.get("skip_logic")
+      case step["type"] do
+        "numeric" ->
+          value = String.to_integer(reply_value)
+          step["ranges"]
+          |> Enum.find_value(nil, fn (range) ->
+            if (range["from"] == nil || range["from"] <= value) && (range["to"] >=
+              value || range["to"] == nil), do: range["skip_logic"], else: false
+          end)
+        "multiple-choice" ->
+          step
+          |> Map.get("choices")
+          |> Enum.find(fn choice -> choice["value"] == reply_value end)
```

Comment:
This will never happen because [here](https://github.com/instedd/ask/blob/feature/322-skip-logic-numeric-steps/lib/ask/runtime/flow.ex#L106) we assign `nil` to the `reply_value` if there isn't any choice with the response of the respondent and then [here](https://github.com/instedd/ask/blob/feature/322-skip-logic-numeric-steps/lib/ask/runtime/flow.ex#L79) we jump to the next question and never call `next_step_by_skip_logic/3` if the reply value is `nil`.